production/jaeger-production-template.yml

```diff
@@ -33,6 +33,11 @@ parameters:
   name: JAEGER_ZIPKIN_SERVICE_NAME
   required: true
   value: zipkin
+- description: The storage type for Jaeger. Possible values are "cassandra", "elasticsearch" or "memory"
+  displayName: Jaeger Storage Type
+  name: JAEGER_STORAGE_TYPE
```

Comment:
maybe a little consistency in env names wouldn't hurt. Just a storage type would be fine. We should also rename JAEGER_ZIPKIN_SERVICE_NAME to ZIPKIN_SERVICE_NAME

Reply:
All vars are prefixed with `JAEGER`, except `IMAGE_VERSION` and `KEYSPACE`. Isn't it better to prefix the others?

Reply:
probably yes, as `IMAGE_VERSION` in C* template looks a bit weird I thought it's a C* version